src/adapters/LayerZeroAdapter.sol

```diff
@@ -20,9 +20,9 @@ import {MathLib} from "../misc/libraries/MathLib.sol";
 
 import {IMessageHandler} from "../common/interfaces/IMessageHandler.sol";
 
-/// @title  LayerZero Adapter
+/// @title LayerZero Adapter
 /// @notice Routing contract that integrates with LayerZero V2.
-/// @dev    A delegate is set on deployment, to configure the DVN and executor
```

Comment:
Can we disable this? I think the old formatting is better

Reply:
Done in [fd108b8](https://github.com/centrifuge/protocol-v3/pull/632/commits/fd108b85774136d39d986838db939c0c37bf9cca)